stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -68,6 +68,7 @@ fn advance_to_nakamoto(peer: &mut TestPeer) {
                 1_000_000_000_000_000_000,
                 PoxAddress::from_legacy(AddressHashMode::SerializeP2PKH, addr.bytes.clone()),
                 12,
+                Point::default(),
```

Comment:
This should be a `StacksPublicKey`.  While both `wsts::Point` and `StacksPublicKey` (an alias to `Secp256k1PublicKey`) are technically points on the secp256k1 curve, we conventionally use `StacksPublicKey` for this (i.e. to indicate that DKG and signing round messages will be signed with ECDSA)

Reply:
Unrelated to this PR, do we have a task in the pipleine to replace secp256k1.rs with Joey's libraries? It would remove the need for this back and forthing/they wrap the same underlying data :D